doc/content/source/bcs/MFEMVectorBoundaryIntegratedBC.md

```diff
@@ -0,0 +1,25 @@
+# MFEMVectorBoundaryIntegratedBC
+
+## Summary
+
+!syntax description /BCs/MFEMVectorBoundaryIntegratedBC
+
+## Overview
+
+Adds the boundary integrator for integrating the linear form
+
+!equation
+(\vec f, \vec v)_{\partial\Omega} \,\,\, \forall \vec v \in V
+
+where $v \in \vec H^1$, $\vec f$ is a vector coefficient of the same dimension, and
+$\hat n$ is the outward facing unit normal vector on the boundary.
```

Comment:
$\hat n$ doesn't actually seem to be used in this file.

Reply:
Indeed - removed